src/powershell/Public/Register-FinOpsHubProviders.ps1

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+<#
+    .SYNOPSIS
+        Registers Azure resource providers required for FinOps hub.
```

Comment:
Align the text with the label. Same for the others.
```suggestion
    .SYNOPSIS
    Registers Azure resource providers required for FinOps hub.
```

Reply:
made changes requested